app/WebParsing/UtsgJsonParser.hs

```diff
@@ -1,26 +1,46 @@
 module WebParsing.UtsgJsonParser
      (getAllCourses,
-      getOrgs,
+      parseAllCourses,
       insertAllMeetings) where
 
 import Config (databasePath, orgApiUrl, timetableApiUrl)
 import Control.Monad.IO.Class (liftIO)
-import Data.Aeson (FromJSON (parseJSON), Object, Value (..), decode, (.!=), (.:?))
+import Data.Aeson (FromJSON (parseJSON), Object, Value (..), decode, decodeFileStrict, (.!=), (.:?))
 import Data.Aeson.Key (toText)
 import Data.Aeson.KeyMap as KM hiding (insert, map)
 import qualified Data.HashMap.Strict as HM
 import Data.Maybe (catMaybes)
 import qualified Data.Text as T
-import Database.Persist.Sqlite (SqlPersistM, insert, insertMany_, runSqlite, selectFirst, (==.))
+import Database.Persist
```

Comment:
I didn't notice this last time, but in general we avoid a "bare import", which imports everything from the module, and instead prefer explicitly listing the functions/types that are imported. Please do that for this line, it'll make the future code easier to maintain.